recipe/build.sh

```diff
@@ -1,43 +1,85 @@
 #!/bin/bash
 
-# install using pip from the whl files on PyPI
-
-if [ `uname` == Darwin ]; then
-    if [ "$PY_VER" == "3.7" ]; then
-        WHL_FILE=https://pypi.org/packages/cp37/c/catboost/catboost-${PKG_VERSION}-cp37-cp37m-macosx_11_0_universal2.whl
-    elif [ "$PY_VER" == "3.8" ]; then
-        WHL_FILE=https://pypi.org/packages/cp38/c/catboost/catboost-${PKG_VERSION}-cp38-cp38-macosx_11_0_universal2.whl
-    elif [ "$PY_VER" == "3.9" ]; then
-        WHL_FILE=https://pypi.org/packages/cp39/c/catboost/catboost-${PKG_VERSION}-cp39-cp39-macosx_11_0_universal2.whl
-    elif [ "$PY_VER" == "3.10" ]; then
-        WHL_FILE=https://pypi.org/packages/cp310/c/catboost/catboost-${PKG_VERSION}-cp310-cp310-macosx_11_0_universal2.whl
-    elif [ "$PY_VER" == "3.11" ]; then
-        WHL_FILE=https://pypi.org/packages/cp311/c/catboost/catboost-${PKG_VERSION}-cp311-cp311-macosx_11_0_universal2.whl
-    elif [ "$PY_VER" == "3.12" ]; then
-        WHL_FILE=https://pypi.org/packages/cp312/c/catboost/catboost-${PKG_VERSION}-cp312-cp312-macosx_11_0_universal2.whl
-    fi
+mkdir bin
+ln -sf ${BUILD_PREFIX}/bin/{swig,ragel,yasm} bin/
+
+if [[ "$target_platform" == "linux-"* ]]; then
+  ln -sf $BUILD_PREFIX/bin/clang $BUILD_PREFIX/bin/${BUILD}-clang++
```

Comment:
This is needed as we're using an older clang here, right?

Reply:
Yes